third_party/compilerworks/README.md

```diff
@@ -0,0 +1,36 @@
+# Using Compilerworks UDFs in BigQuery
+
+## Before You Start
+
+In order to create and use the UDFs within your BigQuery workstreams, ensure that the correct IAM permissions are set.
+
+
+* For creators of the UDFs ensure that they have the following permissions in the dataset of creation:
+  * bigquery.routines.create
+  * bigquery.jobs.create
+
+* For query users of the UDFs please ensure that they have read access to the dataset where the udfs are stored along with the dataset with the actual data.
```

Comment:
Modify to document that users don't technically need access to the dataset with the actual data. [Authorized UDFs](https://cloud.google.com/bigquery/docs/reference/standard-sql/user-defined-functions#authorized_udf_example) allow users to invoke UDFs which touch datasets to which they don't have access.

Reply:
Done.